tools/packaging/common/envoy_bootstrap.json

```diff
@@ -31,6 +31,12 @@
                   "re2.max_program_size.error_level": 1024
               }
           },
+          {
+            "name": "global config",
+            "static_layer": {
+                "overload.global_downstream_max_connections": 2147483647
```

Comment:
That's a few orders of magnitude more than Envoy can handle. Is the intent here to have it "off" by default?

Reply:
Yes. So that it does not warn